### PR TITLE
Added the table for the useful datasets

### PR DESCRIPTION
**Pull request by David-Shibley**

### CODE REVIEW

File: src/markdown/ndoch_2016.md [hunk 1]
```diff
@@ -50,6 +50,14 @@ ___
 - [Open Colorado](http://opencolorado.org/)
 - [data.gov](https://catalog.data.gov)
 
+**Helpful Datasets**
```
Comment (wdoug) [inline src/markdown/ndoch_2016.md]: How do you feel about "Relevant Datasets"?

File: src/markdown/ndoch_2016.md [hunk 2]
```diff
@@ -50,6 +50,14 @@ ___
...
+
+| Dataset Link | Description of Dataset | DataEngine ID
+| ------------ | ---------------------- | -------------
+| [Rent as % of income](http://codataengine.org/find/2010-2014acs-5-year-colorado-gross-rent-percentage-household-income-past-12-months-census-block) | How much people spend on rent relative to their income | 222099
+| [2010](http://codataengine.org/find/2010-metro-denver-federally-subsidized-affordable-housing), [2014](http://codataengine.org/find/2014-metro-denver-federally-subsidized-affordable-housing) | Federally Subsidized Affordable Housing | 222088, 222089
```
Comment (wdoug) [inline src/markdown/ndoch_2016.md]: If we do it like this, should the 'Description of Dataset' be the first column?

Comment (David-Shibley) [inline src/markdown/ndoch_2016.md]: I am not sure, I debated that myself. Tried it both ways and decided to me it made more sense to have the link first and then the description, though it is an easy change if you would like it that way.

File: src/markdown/ndoch_2016.md (hunk 1 above)
Comment (David-Shibley) [inline src/markdown/ndoch_2016.md]: Works for me

**Pull request merged**